ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/SubmitterInfo.scala

```diff
@@ -29,6 +30,7 @@ import com.daml.logging.entries.{LoggingValue, ToLoggingValue}
   */
 final case class SubmitterInfo(
     actAs: List[Ref.Party],
+    readAs: List[Ref.Party],
```

Comment:
Adding this to `SubmitterInfo` but not to `CompletionInfo` looks like the right thing. Then we have the info available in the `WriteService`, but we don't have to propagate it to the `ReadService`.

FYI @oliverse-da This means that we can silently drop the `readAs` field in Canton when we convert this into Canton's `SubmitterMetadata`.